css/main.css

```diff
@@ -1,3 +1,49 @@
 /* Add your custom CSS here */
 
-/* Dont' forget to link this file to your HTML in the <head> */
```

Comment:
`italic` isn't a good class name; it's not only italic, but the font family and size are different.